app/src/main/java/com/vitorpamplona/amethyst/ui/screen/loggedIn/AccountBackupDialog.kt

```diff
@@ -105,10 +113,17 @@ private fun NSecCopyButton(
     val context = LocalContext.current
     val scope = rememberCoroutineScope()
 
+    val keyguardLauncher =
+        rememberLauncherForActivityResult(ActivityResultContracts.StartActivityForResult()) { result: ActivityResult ->
+            if (result.resultCode == Activity.RESULT_OK) {
+                copyNSec(context, scope, account, clipboardManager)
+            }
+        }

```

Comment:
Use the old-fashioned `StartActivityForResult` and pass a `KeyguardManager` intent on older devices.